proto/checksum.proto

```diff
@@ -20,14 +20,20 @@ enum ChecksumAlgorithm {
     Crc64_Xor = 0;
 }
 
+message ChecksumRewriteRule {
+    optional bytes old_prefix = 1;
+    optional bytes new_prefix = 2;
+}
+
 message ChecksumRequest {
     optional uint64 start_ts = 1 [(gogoproto.nullable) = false];
     optional ChecksumScanOn scan_on = 2 [(gogoproto.nullable) = false];
     optional ChecksumAlgorithm algorithm = 3 [(gogoproto.nullable) = false];
+    optional ChecksumRewriteRule rule = 4;
```

Comment:
Is one rule enough, or do we need to pass multiple rules 🤔

Reply:
Currently, one rule is enough, we send checksum request table by table, if we need multiple rules in the future, I think we can add optional field in `ChecksumRewriteRule`, indicate which kind of multiple rules, e.g. `many to one` or `one to many`，it seems more flexible